docs/pages/tokens/border.tsx

```diff
@@ -15,13 +15,23 @@ export default function TokensBorderWidthPage() {
 			>
 				<Prose>
 					<h2>Border Width</h2>
-					<p>These tokens are used to set the width of borders.</p>
+					<p>
+						The following 4 tokens can be used to set the thickness of borders.
```

Comment:
```suggestion
						The following {...} tokens can be used to set the thickness of borders.
```

Can we dynamically generate this number using our tokens object?